certbot/plugins/webroot.py

```diff
@@ -73,7 +79,12 @@ def perform(self, achalls):  # pylint: disable=missing-docstring
 
         self._create_challenge_dirs()
 
-        return [self._perform_single(achall) for achall in achalls]
+        responses = [self._perform_single(achall) for achall in achalls]
+
+        if self.conf("delay-auth"):
+            six.moves.input("Press ENTER to continue authentication...")
```

Comment:
Please the [IDisplay](https://github.com/certbot/certbot/blob/master/certbot/interfaces.py#L375) interface for this task.  An example of how to do that is [here](https://github.com/certbot/certbot/blob/master/certbot/plugins/webroot.py#L152) but you will want to set `pause=True`.